lib/components/UploadSettings.jsx

```diff
@@ -26,25 +26,40 @@ var UploadSettings = React.createClass({
     targetId: React.PropTypes.string,
     isUploadInProgress: React.PropTypes.bool
   },
-  render: function() {
-    // we're already doing a check to see if we want to render in App.jsx
-    // but this is an extra measure of protection against trying to render
-    // when we don't have the groups to do so
+
+  availableUploadGroups: function(){
+    //can only upload for yourself
     if (_.isEmpty(this.props.user.uploadGroups) || this.props.user.uploadGroups.length <= 1) {
       return null;
     }
-    var self = this;
 
-    // sort users alpha by full name
-    var sortedGroups = _.sortBy(this.props.user.uploadGroups, function(group) {
-      if(group.profile.patient.isOtherPerson){
+    //only groups we can upload too, e.g. some people simply aren't `patients` and might be setup without data storage
+    var available = _.filter(this.props.user.uploadGroups, function(group) {
+      return _.isEmpty(group.profile.patient) === false;
+    });
+
+    //and now return them sorted them by name
+    return _.sortBy(available, function(group) {
+      if (group.profile.patient.isOtherPerson) {
```

Comment:
In my [testing scenario](https://github.com/tidepool-org/chrome-uploader/pull/120#issuecomment-104797939), which I believe is the same kind of case as we're trying to set up for Jaeb, there is no `isOtherPerson` property on Jane Doe's entry in Mary Doe's `uploadGroups`.

Is the difference between the results we're seeing @jh-bate the fact that you're manipulating account set-ups directly in the database? Because I'm creating accounts through blip locally...